### PR TITLE
Fix Numpad enter submit on textinputs and also new prop to disable su…

### PR DESCRIPTION
**Pull request by YindSoft**
…bmitting

### CODE REVIEW

File: widget/textinput.go [hunk 1]
```diff
@@ -270,6 +273,12 @@ func (o TextInputOptions) TabOrder(to int) TextInputOpt {
 	}
 }
 
+func (o TextInputOptions) SubmitOnEnter(to bool) TextInputOpt {
```
Comment (mcarpenter622) [inline widget/textinput.go]: Can you please rename "to" to something a bit more descriptive and add a comment for the method?

**Review by mcarpenter622 — CHANGES_REQUESTED** [review]

File: widget/textinput.go (hunk 1 above)
Comment (YindSoft) [reply widget/textinput.go]: all functions to set properties have the parameter "to"

Comment (YindSoft) [reply widget/textinput.go]: sorry, not all, I can change it to newValue which would make more sense, and none of these functions have comments as well, I can add it, but I just preserve the format of the rest of the code.

Comment (mcarpenter622) [reply widget/textinput.go]: Hey, no judgement, but while people are there, I am trying to get a little cleaner code =). I think the only one with "to" would be TabOrder but even that "to" is a bad name.

**Review by mcarpenter622 — APPROVED** [review]

**Pull request merged**